tests/Doctrine/Tests/ORM/Functional/SchemaTool/DDC3460Test.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\SchemaTool;
+
+use Doctrine\ORM\Tools;
+
+class DDC3460Test extends \Doctrine\Tests\OrmFunctionalTestCase
+{
+    /**
+     * @group DDC-3460
+     */
+    public function testDetectMyISAM()
+    {
+        $conn = $this->_em->getConnection();
+
+        if (strpos($conn->getDriver()->getName(), "mysql") === false) {
+            $this->markTestSkipped('this test is only relevant for MySQL');
+        }
+
+        $conn->exec('CREATE TABLE `tweet_user` (id INT NOT NULL AUTO_INCREMENT, name VARCHAR(255) NOT NULL, PRIMARY KEY(id)) ENGINE = MyISAM');
```

Comment:
These models are already in use somewhere else - this makes the test a bit fragile...